FEM/burgers.h

```diff
@@ -0,0 +1,34 @@
+#ifndef BURGERS_H
+#define BURGERS_H
+
+#include <cfloat>
+#include "Eigen/Dense"
+// FEM-Makros
+#include "makros.h"
+#include "tools.h"
+#include "invariants.h"
+
+namespace Burgers
+{
+typedef Eigen::Matrix<double, 6, 1> KVec;
+
+class SolidBurgers
+{
+public:
+	SolidBurgers(const Math_Group::Matrix* data);
```

Comment:
not important: in ogs6 `explicit` keyword should be given to any constructor having a single argument